hack/scripts/sync-prow.sh

```diff
@@ -22,35 +22,43 @@ gh_username="${1}"
 old_prow="$PWD/old_prow"
 new_prow="$PWD/prow"
 
+rm -rf $old_prow $new_prow
 # Clone the old prow repo.
 git clone https://github.com/kubernetes/test-infra.git $old_prow || true
 
 # Clone the new prow repo (this should be your fork).
 git clone "https://github.com/${gh_username}/prow.git" $new_prow || true
 
-# Strip down the master branch of $old_prow to only have commits that touch the prow related directories.
-cd "$old_prow"
-git-filter-repo \
+# These are the --path arguments that define the set of paths we want to migrate:
+path_args="
   --path prow \
```

Comment:
The gopherage paths that Simon wanted to add earlier are not here because we took care of that in k/t-i. Makes sense.